runtime/zeitgeist/src/parameters.rs

```diff
@@ -159,16 +156,18 @@ parameter_types! {
     // 2_678_400_000 = 31 days.
     /// Maximum number of milliseconds a Rikiddo market can be in subsidy gathering phase.
     pub const MaxSubsidyPeriod: Moment = 2_678_400_000;
+    /// Minimum number od blocks for dispute duration, user can not specify value smaller than
+    /// this.
+    pub const MinDisputePeriod: u32 = MIN_DISPUTE_PERIOD;
     // Requirements: MaxPeriod + ReportingPeriod + MaxDisputes * DisputePeriod < u64::MAX.
+    // Note: ReportingPeriod and DisputePeriod are specified in create_market() extrinsic.
     /// The maximum market period.
```

Comment:
You mean that https://github.com/zeitgeistpm/zeitgeist/blob/issue-730/zrml/prediction-markets/src/lib.rs#L587-L595? 

So we have the requirement, that `MaxPeriod` + `oracle_delay` + `oracle_duration` + `MaxDisputes` * `dispute_duration` < u64::MAX.
**I suggest, that we have a maximum for the three deadline parameters too!** So, Mr. Malicious define a market which "never" ends or it comes to some kind of overflow of `u64`.